_docs/tidepool-api/manage-user.md

```diff
@@ -10,6 +10,71 @@ For authentication, maintaining session token and logging out, see the introduct
 
 ## User
 
+### Signup user
+
+There are two parts creating a "user object", the **account** and then the **profile**.
+
+First, create the **account**:
+
+```
+POST /auth/user
+```
+
+Example Request:
+```
+curl -X POST -H "Content-Type: application/json" -d '{"username":"mary@example.com", "password":"myn3wpa55ord", "emails":["mary@example.com"]}'' '<api-endpoint>/auth/user'
+```
+
+Example Response:
+```
+201 Created
+```
+
+```json
+{
+  "userid": "b816f97ec5",
+  "username": "mary@example.com",
+  "emails": [
+    "mary@example.com"
+  ],
+  "termsAccepted": ""
+}
+```
+
+```
+-H "x-tidepool-session-token" : "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJkdXIiOjIuNTkyZSswNiwiZXhwIjoxNDQ4NTY3NDMwLCJzdnIiOiJubyIsInVzciI6IjMzZGYwNTJhNGIifQ.9-5Tf-vMXEtSl9EhWwHq_yAR9k2hCaEv0E3Akd0DX3w"
+```
+
+Then, create the **profile**:
+
+```
+POST /metadata/:userid/profile
+x-tidepool-session-token: <token>
+```
+
+Example Request:
+```
+curl -X POST -H "Content-Type: application/json" -H "x-tidepool-session-token: <token>" -d '{"fullName": "Mary Smith", "shortName":"Mary", "patient": { "birthday": "1990-01-31", "diagnosisDate": "1999-01-31", "aboutMe": "I like oranges"}}' '<api-endpoint>/metadata/<userid>/profile'
```

Comment:
Drop the "aboveMe".  I'd like to lock down our schema so we don't get a bunch of stuff we haven't approved.  Our docs should eventually detail where custom data like this should go.